javascript/javascript2/week1/main.js

```diff
@@ -44,3 +44,43 @@ const danishString = "Jeg har en blå bil";
 console.log(getWordAnddanishLetterCount(danishString));
 const danishString2 = "Blå grød med røde bær";
 console.log(getWordAnddanishLetterCount(danishString2));
+
+const spiritAnimalNames = [
+  "Adventurous Wolf",
+  "Mystical Tiger",
+  "Passionate Leopard",
+  "Intuitive Elephant",
+  "Amazing Unicorn",
+  "Victorious Butterfly",
+  "Magical Monkey",
+  "Wild Dragon",
+  "Creative Dolphin",
+  "Courageous Panther",
+  "Exciting Dove",
+  "Speedy Eagle",
+];
```

Comment:
Really like your choice of spirit animal names! I definitely am aiming to get the 'Wild Dragon' 😎🙏

Reply:
This set of names actually based on first letter of the name and birth month I combined the names ... and wrote as per the homework question. As your name starts with L you will be amazing something. you are amazing!! :-)